lib/xclarity_client/configuration.rb

```diff
@@ -29,8 +33,13 @@ def self.default
         :username   => ENV['LXCA_USERNAME'],
         :password   => ENV['LXCA_PASSWORD'],
         :host       => ENV['LXCA_HOST'],
+        :port       => ENV['LXCA_PORT'],
         :verify_ssl => ENV['LXCA_VERIFY_SSL'] != "NONE"
       })
     end
+
+    def is_integer?(param)
+      param.to_i.to_s == param.to_s
```

Comment:
Suggestion that can be used to test exactly if param is  a valid TCP/IP port: `^([0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$`